Assets/Lighting/Scripts/LightFlicker.cs

```diff
@@ -0,0 +1,92 @@
+using System.Collections.Generic;
+using UnityEngine.Rendering.Universal;
+using UnityEngine;
+
+public class LightFlicker : MonoBehaviour
+{
+    // Start is called before the first frame update
+    private List<Light2D> lights = new List<Light2D>();
+    private int nextUpdate = 1;
+    private int currentFrameCount = 0;
+    private bool innerLightOn = true;
+    private bool outerLightOn = true;
+    private bool isInnersTurn = false;
+    private bool isOutersTurn = true;
+
+    public float flickerLowerFrequency;
+    public float flickerUpperFrequency;
+
+    void Start()
+    {
+        GameObject[] flickers = GameObject.FindGameObjectsWithTag("Flicker");
+        foreach(GameObject gameObject in flickers)
```

Comment:
In the future try not to use foreach loops with Unity code there is a few reasons why but it's generally not a good idea.

Reply:
I'm assuming performance?